data/historical-benchmarks/demo.csv

```diff
@@ -0,0 +1,689 @@
+Table 2: MIPS Benchmark Results ,,,,,,,,,,,,,

```

Comment:
Wondering if this is just for demo purposes it makes sense to store this whole file. I think it's good to have an example file for testing purposes but maybe we want to just have a few of each scenario we want to cover in testing. Of course we'll generate the benchmarks using all the data but don't think all this original data needs to be stored here.